bot/cogs/Calculator.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import re
+import discord
```

Comment:
PEP8 conventions state there should be a newline between python imports and third party library imports, see clem_bot.py for an example